Makefile

```
@@ -2,44 +2,39 @@ GO_BIN       ?= go
 
-print-cli-name: ## Print CLI name for CI/CD usage
+print-cli-name:
 	@echo "$(CLI_NAME)"
 
-release: build ## Build and create GitHub release
-	@echo "Creating release for $(VERSION)"
-	mkdir -p $(BUILD_DIR)
-	gh release create $(VERSION) \
-		--title "Release $(VERSION)" \
-		--notes "Release notes for $(VERSION)" \
+release: build
```

checkmake (0.3.2)

[warning] Target body for "release" exceeds allowed length of 5 lines (9).

(maxbodylength)